docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -205,12 +217,21 @@ class RabbitMqConfig extends SprykerRabbitMqConfig
 }
 ```
 
+In the example, we use `Pyz` prefix, but you can use any other prefix, for example your project name. After the fix please re-evaluate the code. The same error shouldn’t be returned.
+
 ---
 
+## NotUnique:Constant
 
-## Constant name is not unique
+Constant names must be unique. For an example of the issue and how to solve it, see the following sections. Usage of this rule will guaranty that your constants won't conflict with the constants that appear in Spryker core in the future.
 
-Constant names must be unique. For an example of the issue and how to solve it, see the following sections.
+### Example of error in the Evaluator output: Constant name is not unique
+
+```bash
+------------------ ---------------------------------------------------------------------------------------------------------------
+NotUnique:Constant "Pyz\Client\RabbitMq\RabbitMqConfig::CUSTOM_CONST" name has to have project namespace, like "PYZ_CUSTOM_CONST".
```

Comment:
This is not in line with the other examples. We should be consistent, and use either placeholders or variables in all the code samples. As we already have real examples in other samples, add it here as well.

Reply:
I don't get this comment. I other examples we have also the examples of the output that relate to the code example with the problem.